docs/docusaurus.config.js

```diff
@@ -110,6 +115,11 @@ const config = {
         hideOnScroll: false,
         items: [
           // right navbar items
+          {
+            label: "Home",
+            position: "right",
+            to: "/",
+          },
```

Comment:
Maybe I should remove this until the home page is totally ready to deploy. 

My original thinking was that I would do build the Nebari landing page in two PRs. The first PR puts up a rough draft. The second PR polishes it. But it seems like we're going to fix everything in this PR before we merge it in? How would you prefer to go about this @trallard?

Reply:
I thought we had agreed to fix the background images + footer + dark/light colour inconsistencies in subsequent PRs to get this merged ASAP.

The one outstanding item would be to replace the mock dashboard #212 has discussions about this, and @pavithraes  and I can sync about this tomorrow morning.

Right now, the images for the integrations - which was the main blocking for me seem to be working somehow. (tested in Brave - private and not, Safari and Chrome or maybe I got lucky).

Reply:
edit your `img` fix did the trick